kili/cli.py

```diff
@@ -12,8 +17,108 @@ def cli():
     """Kili Command line interface."""
 
 
+@cli.group(context_settings=CONTEXT_SETTINGS)
+def project():
+    """Commands to interact with a Kili project"""
+
+
+@project.command(name='create')
+@click.option('--api-key', type=str, envvar='KILI_API_KEY', required=True,
+              help='Your Api Key')
+@click.option('--interface', type=click.Path(exists=True), required=True,
+              help='Path pointing to your json interface file')
+@click.option('--title', type=str, required=True,
+              help='Title of your project')
+@click.option('--input-type', type=click.Choice(INPUT_TYPE), required=True,
+              help='type of data in your project')
```

Comment:
Project input data type

Reply:
and add examples of `INPUT_TYPE`

Reply:
When using click.choice, the -help option automatically display all possible choices
<img width="658" alt="Capture d’écran 2022-06-13 à 17 22 58" src="https://user-images.githubusercontent.com/82399630/173388283-5ba956c6-c342-4fd7-b8a6-f18921fe1370.png">